org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/nexus/SolsticeScanMonitor.java

```diff
@@ -233,8 +250,8 @@ public void scanFinished() throws ScanningException {
 			throw new ScanningException("Could not write scanFinished to NeXus file", e);
 		}
 		
-		Duration scanDurationObj = Duration.between(scanStartTime, Instant.now());
-		String scanDurationStr = durationInMillisToString(scanDurationObj);
+		Duration scanDuration = Duration.between(scanStartTime, Instant.now());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "scanDuration" which hides the field declared at line 97. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AHiddenFieldCheck)